web/src/pages/Docs.vue

```diff
@@ -62,18 +63,25 @@ export default {
   },
   methods: {
     onChange() {
-      const location = document.getElementById("docs")
-        .contentWindow.location.href
-      if (location) {
-        this.load(location)
+      const docsFrame = document.getElementById("docs")
+      docsFrame.contentWindow.document.querySelectorAll('a').forEach((a) => {
+        if (!a.href.startsWith(location.origin)) {
+          // open all foreign links in a new tab
+          a.setAttribute('target', '_blank')
```

Comment:
do we need to rewrite the links or could we rewrite the target after it it clicked?

Reply:
This is the simplest approach I found, the problem with the other approach is AFAIK that you could also get a trigger when you click an element inside an a tag ...

Reply:
I see then we keep it like that until we have actual issues